modules/imgproc/src/pyramids.cpp

```diff
@@ -1266,6 +1266,8 @@ static bool ocl_pyrUp( InputArray _src, OutputArray _dst, const Size& _dsz, int
 
 }
 
+// TODO: enable it when the corresponding IPP code is fixed
```

Comment:
Need to describe the issue, or, better, create a ticket and add link here.

Reply:
Opened an issue: #25976
Since there's no corresponding functions in ippicv, I propose to remove this implementation.
It's easier to write it from scratch than fixing this code that looks like it's in a middle of refactoring.